packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -387,10 +386,9 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
                 const hostSnapshotOptions = this.hostPolicy.snapshotOptions;
                 let cachedSnapshot: IOdspSnapshot | undefined;
-                const usePost = !!this.hostPolicy.usePostForTreesLatest;
```

Comment:
Is this already true today? I.e. do we have 100% of traffic going through POST and we see no issues with that?

Reply:
Yes, I don't see any error on POST in telemetry for some time now. No need of fallback now.